sentry-compose/src/androidMain/kotlin/io/sentry/compose/SentryNavigationIntegration.kt

```diff
@@ -73,3 +75,9 @@ public fun NavHostController.withSentryObservableEffect(
     }
     return this
 }
+
+@Composable
+public fun rememberNavController(vararg navigators: Navigator<out NavDestination>): NavHostController {
```

Comment:
How about we have a singleton object, like `SentryComposeInstrumentation` where we'd put this method? Now it has the same signature as the original `rememberNavController`, i think the users might get confused when picking the right one from autocomplete.